packages/stream-server/README.md

```diff
@@ -0,0 +1,56 @@
+# Meteor's DDP stream server based on SockJS
+
+> Network core of the Meteor's DDP.
+
+The `stream-server` package provides:
+
+* the default Meteor's stream server based on SockJS;
+* the mechanism to replace bundled `StreamServer`.
+
+## How to add your custom `StreamServer` to `StreamServers` and activate it
+
+For example your package has name `custom-stream-server`.
+
+1. Enable access to the `StreamServers` variable.
+
+    `custom-stream-server/package.js`
+
+    ```js
+    Package.onUse(function (api) {
+      // Add following lines
+      api.use(['stream-server'], 'server');
+      api.imply(['stream-server'], 'server');
+      api.export('StreamServers', 'server');
+      // ...
+    });
+    ```
+
+2. Push `CustomStreamServer` to the `StreamServers` array.
+
+    `custom-stream-server/server.js`
+
+    ```js
+    class CustomStreamServer {
+      constructor() {
+        // this.server = new WebSocket.Server();
+      }
+    }
+    
+    StreamServers.push(CustomStreamServer);
+    ```
+
+3. Activate your custom stream server.
+
+    In the `.meteor/packages` place your package name before packages:
+    
+    * `meteor-tools`
+    * `ddp`
+    * `ddp-server`
+
+    Example of the `.meteor/packages`
+    
+    ```
+    custom-stream-server
+    meteor-tools
+    # other packages
```

Comment:
For example, this recommendation might become unnecessary if we can actively let the other packages know when a new stream server has been added, rather than just relying on the load order of the packages.

Reply:
I don't found the way how to change the DDP `Server.stream_server` after the DDP has initialized.